src/main/scala/cromwell/binding/WdlImplicits.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.binding
+
+import cromwell.binding.types.{WdlIntegerType, WdlStringType}
+
+object WdlImplicits {
+  implicit class WdlString(val string: String) extends AnyVal {
```

Comment:
Another possibility would be to make a typeclass for things which are toWdlValue-able although I'd probabably leave it as is until n > 2.